themes/light/ice.css

```
@@ -0,0 +1,146 @@
+/* [THEME META] {"ver":1, "author":"donut28", "description":"Blue but with Discord light mode mixed in. Will make you blind."} */

```

Making a theme bold does not make it accessible (According to WebAIM). While gradients do help visually, they are not pronounced enough here (and there are some almost-solid colors failing AA). Please opt-out, or update theme.
```suggestion
/* [THEME META] {"ver":1, "author":"donut28", "description":"Blue but with Discord light mode mixed in. Will make you blind.", "accessibility","ignore"} */
```